react-native-libraries.json

```diff
@@ -13330,5 +13330,17 @@
     "ios": true,
     "android": true,
     "newArchitecture": false
+  },
+  {
+    "githubUrl": "https://github.com/reduxjs/redux-thunk",
+    "examples": [
+      "https://github.com/reduxjs/redux-toolkit/tree/HEAD/examples/publish-ci/react-native",
+      "https://github.com/reduxjs/redux-toolkit/tree/HEAD/examples/publish-ci/expo"
+    ],
```

Comment:
`redux-thunk` is not a directly imported package or the examples below, besides that LGTM! 👍 

```suggestion
```